src/community/blocks.tsx

```diff
@@ -0,0 +1,277 @@
+                    {secondaryValue}
+                </Text2>
+                <Text5 color={valueColor}>{value}</Text5>
+            </div>
+        </Inline>
+    );
+};
+
+interface HighlightedValueBlockProps {
+    headline?: RendersNullableElement<typeof Tag>;
+
+    mainHeading: {
+        value: number;
+        text: string;
+    };
+
+    secondHeading?: {
+        value: number;
+        text: string;
+    };
+
+    secondaryValue?: number;
+
+    title?: string;
+    description?: Array<string>;
+
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    headline,
+    mainHeading,
+    secondHeading,
+    secondaryValue,
+    title,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            {headline && <Box paddingBottom={24}>{headline}</Box>}
+
+            <Stack space={2}>
+                <Inline space={8} alignItems="baseline">
+                    <Text8 color={valueColor}>{mainHeading.value}</Text8>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {mainHeading.text}
+                    </Text2>
+                </Inline>
+                {secondHeading && (
+                    <Inline space={8} alignItems="baseline">
+                        <Text8 color={valueColor}>{secondHeading.value}</Text8>
+                        <Text2 regular color={vars.colors.textSecondary}>
+                            {secondHeading.text}
+                        </Text2>
+                    </Inline>
+                )}
+                {secondaryValue && <Text8 color={vars.colors.textSecondary}>{secondaryValue}</Text8>}
+            </Stack>
+            <Box paddingTop={8}>
+                <BlockContent title={title} description={description} />
+            </Box>
+        </div>
+    );
+};
+
+interface ValueBlockProps {
+    title?: string;
+    value?: string;
+    description?: Array<string>;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const ValueBlock: React.FC<ValueBlockProps> = ({
+    title,
+    value,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel} className={styles.column}>
+            <Text2 regular color={vars.colors.textPrimary}>
+                {title}
+            </Text2>
+            <Text8 color={valueColor}>{value}</Text8>
+            <BlockContent description={description} />
+        </div>
+    );
+};
+
+interface ProgressBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+
+    progressPercent?: number;
+    reverse?: boolean;
+
+    heading: {
+        value: number;
+        text: string;
+    };
+
+    description?: string;
+
+    valueColor?: string;
```

Comment:
shouldn't this be inside heading?
```ts
heading: {
    value: number;
    valueColor?: string;
    text: string;
}
```

Reply:
I don't think so, as it could have more than one description and only one header and thus the prop color would not allow the values ​​of the two headers to be of different colors

<img width="136" alt="image" src="https://github.com/Telefonica/mistica-web/assets/123959276/a064eed1-1345-4c4c-a4ed-5b6e8c97adeb">